integrations/wiz/wiz/client.py

```diff
@@ -1,7 +1,13 @@
+import codecs
+import csv
+import json
+import time
+from contextlib import closing
 from enum import StrEnum
-from typing import Any, AsyncGenerator
+from typing import AsyncGenerator, Dict, List, Any
 
 import httpx
+import requests
```

Comment:
Is there a reason why we're using requests over httpx? If you want a sync function, httpx provides a sync client also

Reply:
no, i have changed it now